src/trading/utils.ts

```diff
@@ -90,3 +91,19 @@ export function getTradeParametersAfterQuote({
 }): TradeParameters {
   return { ...quoteParameters, sellToken: orderParameters.sellToken }
 }
+
+/**
+ * ETH-flow orders are special and need to be adjusted
+ * 1. Sell token should be the wrapped native currency
+ * 2. Default slippage is 2%
+ */
+export function adjustEthFlowOrderParams(
+  chainId: SupportedChainId,
+  params: TradeParameters | LimitTradeParametersFromQuote
+): typeof params {
+  return {
+    ...params,
+    sellToken: WRAPPED_NATIVE_CURRENCIES[chainId],
+    slippageBps: typeof params.slippageBps === 'number' ? params.slippageBps : ETH_FLOW_DEFAULT_SLIPPAGE_BPS,
```

Comment:
Here users will always have 2% for EthFlow, even if they set more than that.
I would expect the minimum to be applied only if value is < than min.

Reply:
Sorry, I don't understand.
This code takes `slippageBps` from params and fallback to `ETH_FLOW_DEFAULT_SLIPPAGE_BPS` if it's not set.

Reply:
Sorry, I don't this is a good idea :) 
For SDK users this will be unexpected behaviour.
As a SDK user I would expect to have exactly that slippage that I set.
Otherwise, it will be impossible to set for example 30 BPS because default slippages is 50, but I would like to have exactly 30 BPS.